pkg/core/native/crypto_blspoints.go

```diff
@@ -0,0 +1,84 @@
+package native
+
+import (
+	"fmt"
+
+	bls12381 "github.com/consensys/gnark-crypto/ecc/bls12-381"
+	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
+)
+
+// Wrappers around bls12381 types that must be used as stackitem.Interop values and
+// implement stackitem.Equatable interface.
+type (
+	G1Affine struct {
+		*bls12381.G1Affine
+	}
+	G1Jac struct {
+		*bls12381.G1Jac
+	}
+	G2Affine struct {
+		*bls12381.G2Affine
+	}
+	G2Jac struct {
+		*bls12381.G2Jac
+	}
+	GT struct {
+		*bls12381.GT
+	}
+)
+
+var (
+	_ = stackitem.Equatable(&G1Affine{})
+	_ = stackitem.Equatable(&G1Jac{})
+	_ = stackitem.Equatable(&G2Affine{})
+	_ = stackitem.Equatable(&G2Jac{})
+	_ = stackitem.Equatable(&GT{})
+)
+
+// Equals implements stackitem.Equatable interface.
+func (a *G1Affine) Equals(other any) bool {
+	b, ok := other.(*G1Affine)
+	return ok && a.Equal(b.G1Affine)
+}
+
+// Equals implements stackitem.Equatable interface.
+func (a *G1Jac) Equals(other any) bool {
+	b, ok := other.(*G1Jac)
+	return ok && a.Equal(b.G1Jac)
+}
+
+// Equals implements stackitem.Equatable interface.
+func (a *G2Affine) Equals(other any) bool {
+	b, ok := other.(*G2Affine)
+	return ok && a.Equal(b.G2Affine)
+}
+
+// Equals implements stackitem.Equatable interface.
+func (a *G2Jac) Equals(other any) bool {
+	b, ok := other.(*G2Jac)
+	return ok && a.Equal(b.G2Jac)
+}
+
+// Equals implements stackitem.Equatable interface.
+func (a *GT) Equals(other any) bool {
+	b, ok := other.(*GT)
+	return ok && a.Equal(b.GT)
+}
+
+// NewEquatable converts provided bls12381 point type to the stackitem.Equatable.
+func NewEquatable(point any) stackitem.Equatable {
+	switch t := point.(type) {
+	case *bls12381.G1Affine:
+		return &G1Affine{t}
```

Comment:
...or use plain structure here (not a pointer).

Reply:
Fixed.